virt/virtual_machines/advanced_vm_management/virt-efi-mode-for-vms.adoc

```diff
@@ -0,0 +1,11 @@
+[id="virt-efi-mode-for-vms"]
+= EFI mode for virutal machines
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-efi-mode-for-vms
+toc::[]
+
+A virtual machine (`VM`) can be booted in `EFI` (Extensible Firmware Interface) mode.
```

Comment:
Passive voice + I do not think the backticks are necessary here. Suggestion: "You can boot a virtual machine (VM) in Extensible Firmware Interface (EFI) mode."